Mathlib/Algebra/Central/End.lean

```diff
@@ -0,0 +1,27 @@
+/-
+Copyright (c) 2025 Monica Omar. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Monica Omar
+-/
+module
+
+public import Mathlib.Algebra.Central.Defs
+public import Mathlib.LinearAlgebra.FreeModule.Basic
+
+/-!
+# `Module.End R M` is a central algebra
+-/
+
+open Module Free
+
+variable {R M : Type*} [CommSemiring R] [AddCommMonoid M] [Module R M] [Free R M]
+
+/-- The center of endomorphisms on a vector space is trivial,
+in other words, it is a central algebra. -/
+public instance Algebra.IsCentral.end : IsCentral R (End R M) where
```

Comment:
I thought the policy was "just use the module boilerplate for now and there will be tools to optimise stuff later"